tests/bluetooth/audio/ascs/src/main.c

```diff
@@ -179,45 +185,6 @@ ZTEST_F(ascs_test_suite, test_release_ase_on_callback_unregister)
 	zassert_equal(0x00, hdr->ase_state, "unexpected ASE_State 0x%02x", hdr->ase_state);
 }
 
-ZTEST_F(ascs_test_suite, test_abort_client_operation_if_callback_not_registered)
-{
-	const struct test_ase_cp_chrc_value_param *param;
-	const struct test_ase_cp_chrc_value_hdr *hdr;
-	const struct bt_gatt_attr *ase_cp = fixture->ase_cp;
-	struct bt_bap_stream *stream = &fixture->stream;
-	struct bt_conn *conn = &fixture->conn;
-	struct bt_gatt_notify_params *notify_params;
-	uint8_t ase_id;
-
-	if (IS_ENABLED(CONFIG_BT_ASCS_ASE_SNK)) {
-		ase_id = fixture->ase_snk.id;
-	} else {
-		ase_id = fixture->ase_src.id;
-	}
-
-	zexpect_not_null(ase_cp);
-	zexpect_true(ase_id != 0x00);
-
-	/* Set ASE to non-idle state */
-	test_ase_control_client_config_codec(conn, ase_id, stream);
-
-	/* Expected ASE Control Point notification with Unspecified Error was sent */
-	expect_bt_gatt_notify_cb_called_once(conn, BT_UUID_ASCS_ASE_CP, ase_cp,
-					     EMPTY, TEST_ASE_CP_CHRC_VALUE_SIZE(1));
-
-	notify_params = mock_bt_gatt_notify_cb_fake.arg1_val;
-	hdr = (void *)notify_params->data;
-	zassert_equal(0x01, hdr->opcode, "unexpected Opcode 0x%02x", hdr->opcode);
-	zassert_equal(0x01, hdr->number_of_ases, "unexpected Number_of_ASEs 0x%02x",
-		      hdr->number_of_ases);
-	param = (void *)hdr->params;
-	zassert_equal(ase_id, param->ase_id, "unexpected ASE_ID 0x%02x", param->ase_id);
-	/* Expect Unspecified Error */
-	zassert_equal(0x0E, param->response_code, "unexpected Response_Code 0x%02x",
-		      param->response_code);
-	zassert_equal(0x00, param->reason, "unexpected Reason 0x%02x", param->reason);
-}
```

Comment:
Why was this test removed?

Reply:
Right, so it would need to re-added if we implemented separate register/unregister functions, right?

Reply:
If there is no dependency on calling 'bt_bap_unicast_server_register_cb' and the proposed ascs function, then yes, this should definitely be re-added :)